data/themes/default/style.css

```diff
@@ -620,15 +620,17 @@ TrackContentObjectView   {
 	qproperty-selectedColor: #006B65;
 	qproperty-BBPatternBackground: #373d48;
 	qproperty-textColor: #fff;
+	qproperty-textBackgroundColor: rgba(0, 0, 0, 75);
 	qproperty-textShadowColor: rgb(0,0,0,200);
```

Comment:
@Umcaruje Commit 822893619e92247bfa01b8461f0afe29cccbd351 adds the ability to render shadows back. The rationale is that if a design does not want to use shadows it can just set the property `qproperty-textShadowColor` to something completely transparent. That way we are more flexible.